docs/docs/artifact-store.md

```diff
@@ -150,25 +143,29 @@ Server: Jetty(8.1.11.v20130520)
 ```
 
 
-## Automatic Artifact Storing
+## Automatic Artifact Storage
+
+An AppDefinition holds a sequence of URIs that Marathon fetches when it starts each app instance.
 
-An AppDefinition holds a sequence of URIs, that get fetched on each instance, that gets started.
 The artifact could be fetched directly from the source, or put into the artifact store.
```

Comment:
That sentence tries to explain the difference between `uris` (which BTW is deprecated and replaced by `fetch`) and storeURLSs. As long as the difference becomes clear, the sentence can be removed.